lib/internal/Magento/Framework/Crontab/CrontabManager.php

```diff
@@ -209,7 +209,7 @@ private function save($content)
             $this->shell->execute('echo "' . $content . '" | crontab -');
         } catch (LocalizedException $e) {
             throw new LocalizedException(
-                new Phrase('Error during saving of crontab: %1', [$e->getPrevious()->getMessage()]),
+                new Phrase('Error during saving of crontab: %1', [$e->getMessage()]),
```

Comment:
You can get the idea of what this code does from a failed unit test: https://testing-service.magento-community.engineering/reports/magento/magento2/pull/24657/c2a63ebfba0bbcbfcf7634d0bfe801ca/Unit/allure-report-ce/index.html#suites/d9d312cf829cbf15ca5c954b5cd9bcd5/1969e26d32677399/

However, it seems in some cases `$e->getPrevious()` is not present - please add such logic and write a new unit test to cover it.